README.md

```diff
@@ -30,21 +47,17 @@ The DynamoDb table has this structure:
 - `processorId`: S - The unique identifier of the processor
 - `startedAt`: N - The datetime the signal has started to be processed
 - `completedAt`: N - The datetime when the signal has been completed
-- `expiresOn`: N - The datetime when the signal process will expires
-
-Each time a processor with a given `processorId` attempt to process a signal identified by `id`, it updates or writes on the table a record with `id`, `processorId`, `startedAt`. If the record with given `id` and `processorId` was already present, its value is returned to the library otherwise nothing is returned. After the process has runned successfully, the library mark it as completed by storing the `completedAt` and the `expiresOn` fields.
-
-The `expiresOn` allows to clean up old data and re-run duplicate after some time.
+- `expiresOn`: N - The datetime when the signal process will expire
 
-When the library attemp to start a process, these scenarios can happen:
+Each time a processor with a given `processorId` attempts to process a signal identified by `id`, it updates or writes on the table a record with `id`, `processorId`, `startedAt`. If a record with the given `id` and `processorId` is already present, its value is returned to the library, otherwise nothing is returned. After the process has run successfully, the library marks it as completed by storing the `completedAt` and the `expiresOn` fields.
 
-1) The signal has never been processed previously (not previous record found)
-2) The signal has been already processed previously (`completedAt` is present)
-3) The signal has timeout processing (`completedAt` is absent and `startedAt` + `processingTime` is in the past)
-4) The signal is stil being processing (`completedAt` is absent and `startedAt` + `processingTime` is in the future)
+The `expiresOn` allows clean up of old data and duplicate re-runs after some time.
 
-In cases (1) and (2) the library allows the signal to be processed. In cases (3) the library does not allow the signal to be processed again and (4) the library wait the process to either complete or timeout before taking any decision.
+When the library attempts to start a process, these scenarios can happen:
 
-## Terraform
+1) The signal has never been processed previously (no previous record found)
+2) The signal has already been processed previously (`completedAt` is present)
+3) The signal has timed out processing (`completedAt` is absent and `startedAt` + `processingTime` is in the past)
+4) The signal is still being processed (`completedAt` is absent and `startedAt` + `processingTime` is in the future)
 
-An [example terraform file](example.tf) is provided for provisioning the backing database with DynamoDB.
\ No newline at end of file
+In cases (1) and (2) the library allows the signal to be processed. In case (3) the library does not allow the signal to be processed again and in (4) the library waits for the process to either complete or timeout before making any decision.
```

Comment:
@l-catallo or @filosganga: Since we're here, are these points correct? 
> In case (3) the library does not allow the signal to be processed again

Case 3 is when the "lock expired" and I thought that the whole reason of this expiration was to allow reprocessing the same signal id when something goes wrong.

Also, for the case 2 we say:
> In cases (1) and (2) the library allows the signal to be processed.
If the `completedAt` is present and we try to process the same signal (same `id`) isn't that a dupicate?

Reply:
Yeah I think case 2 and 3 are swapped.
In case (2) the process is stopped from being executed.
In case (3) the process is executed.

Reply:
@kittsville Would it be asking too much if we asked you to make this little update? 🙏 I'm also personally happy to approve and update it myself after it gets merged.

Reply:
Yeah sure, I'm happy to 👍